components/brave_wallet_ui/components/shared/copy-tooltip/copy-tooltip.tsx

```diff
@@ -22,16 +22,18 @@ type Props = {
   tooltipText?: string
   actionText?: string
   text?: string
+  isConfidential?: boolean
 } & ToolTipProps
```

Comment:
My concern if this field should be mandatory for each <CopyTooltip/>. Would that be ok if misconfigure some of them? cc: @kdenhartog